apps/prairielearn/src/components/InstructorInfoPanel.html.ts

```diff
@@ -166,9 +166,13 @@ function QuestionInfo({
   //
   // In the future, this should use some kind of "allow import" flag on the
   // question so that this behavior can be achieved within other courses.
```

Comment:
I believe this setting would be `"sharePublicly": false, "shareSourcePublicly": true`. Once https://github.com/PrairieLearn/PrairieLearn/pull/10847 lands, we can probably remove this special-casing.

Reply:
Nit: This comment reads funny now with both the old comment above and your new comment below. Because the part above sounds like `share_source_publicly` doesn't exist, and then the part below says it does. Maybe replace the whole thing by.

```
// Don't show the sharing name in the QID if the question is not
// shared publicly for importing, such as if only `share_source_publicly` is set.
// TODO remove special casing of example course once all example course questions are updated with `share_source_publicly`
```

then you just remove the last line of the comment once the other PR is in.

Reply:
I adjusted the comment in df71aa7.